lib/mix/lib/releases/plugins/plugin.ex

```diff
@@ -169,10 +169,12 @@ defmodule Mix.Releases.Plugin do
     plugins =
       for p <- release.profile.plugins do
         case p do
-          {_p, _opts} ->
+          {mod, _opts} ->
+            load(mod)
```

Comment:
I think a better place for this is in the config file - so that we load all plugin modules in one go, rather than hitting the code server for every event which hits all of the plugins. Not super important in Distillery, but it also guarantees that we can count on plugins being available everywhere after that.